python quiz solution.ipynb

```diff
@@ -0,0 +1,190 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "a0f953e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "3c7e43f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w=open('contact.csv','w',newline='')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "2de3f5cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=csv.writer(w)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "18a74f4a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a=csv.writer(w)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "837ab2c5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "header=[\"name\",\"contact\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "74b1a792",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "14"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "a.writerow(header)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "2a0fa579",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data=[['parth',9658743210],\n",
+    "     ['dhrumin',9965848753],\n",
+    "     ['dev',7636895765]]\n",
+    "   "
```

Comment:
is it possible to take input valaue from user?

Reply:
Python allows for user input